app/controllers/licence_controller.rb

```diff
@@ -136,8 +134,8 @@ def local_authority_slug
     return nil unless mapit_response.location_found?
 
     @_la_slug ||= begin
-      tier_override = :county_unitary if LICENCE_SLUGS_WITH_COUNTY_TIER_OVERRIDE.include?(params['slug'])
-      LicenceLocationIdentifier.find_slug(mapit_response.location.areas, artefact, tier_override)
+      # tier_override = :county_unitary if LICENCE_SLUGS_WITH_COUNTY_TIER_OVERRIDE.include?(params['slug'])
```

Comment:
Luckily `LICENCE_SLUGS_WITH_COUNTY_TIER_OVERRIDE` goes away in the next commit I think